frontend/server/src/main/java/org/pytorch/serve/job/Job.java

```diff
@@ -20,6 +22,21 @@ public Job(String modelName, String version, WorkerCommands cmd, RequestInput in
         this.modelVersion = version;
         begin = System.nanoTime();
         scheduled = begin;
+
+        Map<String, String> headers = input.getHeaders();
+        if (headers.containsKey("X-Priority")) {
```

Comment:
Not that it is incorrect, but maybe it is a bit unnatural to use a header instead of the body for the priority? But if it is the only way, then lets keep it.

I am also not sure about the name `X-Priority` - apparently that header is already used in some cases for emails.
In the past, I started using the convention of prefixing headers with `X-TS-` so that it is clear, that it is our own header.

Reply:
That's a good idea, I'll change it to `X-TS-`. I think at this point the request body is not yet parsed, so it has to be a header unless we want introduce overhead at the pre-queueing stage.